src/UI/Internal/Button.elm

```diff
@@ -39,7 +40,12 @@ type Button msg
 
 type ButtonBody
     = BodyText String
-    | BodyIcon Icon
+    | BodyIcon Icon (Maybe BodyIconLabelPosition)
```

Comment:
maybe a 

type ButtonBody 
= BodyText String
| BodyIcon Icon
| BodyIconLeftText Icon
| BodyIconRightText Icon

is better and we avoid the Maybe?